javascript/javascript1/week2/studentmanager.js

```diff
@@ -0,0 +1,38 @@
+const class07Students = [];
+const queen='Margrethe II';
+function addStudentToClass(studentName) {
+  if (!class07Students.includes(queen)){
```

Comment:
The Queen should be added only if someone adds her ie `addStudentToClass("Queen");`